### PR TITLE
fix(wc): update lerna and use workspaces

### PR DESCRIPTION
**Pull request by ariellalgilmore**
### Related Ticket(s)

n/a
### Description

Set `enableTransparentWorkspaces` to false, so that `web-components` doesn't use the local cwc repo, but instead points to the remote cwc registry 

test PR on main double checking it works with ci: https://github.com/carbon-design-system/carbon-for-ibm-dotcom/pull/10130/files

### Changelog

**New**

- {{new thing}}

**Changed**

- enableTransparentWorkspaces = false
- introduce `workspaces:` to some packages
- update lerna to 6.x same as main branch

**Removed**

- {{removed thing}}

### CODE REVIEW

**Review by annawen1 — APPROVED** [review]
LGTM!

**Review by IgnacioBecerra — APPROVED** [review]
LGTM, glad the solution actually worked, we would've gone crazy otherwise haha

File: lerna.json [hunk 1]
```diff
@@ -1,7 +1,6 @@
 {
   "lerna": "3.20.2",
```
Comment (kennylam) [inline lerna.json]: @ariellalgilmore This should be the same as the installed version of `lerna`

Comment (ariellalgilmore) [reply lerna.json]: fixed!

**Pull request merged**